loader.php

```diff
@@ -189,3 +190,110 @@ function enqueue_assets( $directory, $opts = [] ) {
 		wp_enqueue_style( $opts['handle'] );
 	}
 }
+
+/**
+ * Display an overlay error when the React bundle cannot be loaded. It also stops the execution.
+ *
+ * @param array $details
+ */
+function handle_assets_error( $details = [] ) {
```

Comment:
Oh, also, we should `trigger_error( ..., E_USER_WARNING )` in production, and not output internal details.